scispacy/hyponym_detector.py

```diff
@@ -62,6 +62,8 @@ def expand_to_noun_compound(self, token: Token, doc: Doc):
 
         start = token.i
         while True:
+            if start==0:
```

Comment:
Please add spaces between symbols, (our linter requires it), like `if start == 0:`